src/Types/ConversionException.php

```diff
@@ -98,7 +100,7 @@ public static function conversionFailedInvalidType(
      *
      * @return ConversionException
      */
-    public static function conversionFailedSerialization($value, $format, $error)
+    public static function conversionFailedSerialization($value, $format, $error /*, ?Throwable $previous = null */)
```

Comment:
💡 Adding the parameter would be a BC break because the class is not final. I'll add the parameter on the 4.0.x branch.